src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -714,9 +826,15 @@ public void deleteCourseCascade(String courseId) {
         return courseSummaryList;
     }
     
-    public String getCourseStudentListAsCsv(String courseId, String googleId) throws EntityDoesNotExistException {
+    /**
+     * @param courseId
+     * @param instructorGoogleId
+     * @return a CSV for the details(name, email, status) of all students belongs to a given course
```

Comment:
return a CSV string of the details(name, email, status) of all students belonging to a given course